path_login_test.go

```diff
@@ -34,9 +40,188 @@ var (
 	testProjectedUID         = "77c81ad7-1bea-4d94-9ca5-f5d7f3632331"
 	testProjectedMockFactory = mockTokenReviewFactory(testProjectedName, testNamespace, testProjectedUID)
 
-	testDefaultPEMs = []string{testECCert, testRSACert}
+	// generate with: openssl ecparam -genkey -name prime256v1
+	ecdsaPrivateKeyText = `-----BEGIN EC PRIVATE KEY-----
```

Comment:
Annoyingly, test data like this can often trigger false positives in automated scanners. Could we generate the key material dynamically using Go's crypto libraries instead?